tests/integ/modin/series/test_reindex.py

```diff
@@ -259,17 +259,17 @@ def perform_reindex(series):
     )
 
 
-@sql_count_checker(query_count=2, join_count=1)
+@sql_count_checker(query_count=1, join_count=2)
 @pytest.mark.parametrize("limit", [None, 1, 2, 100])
 @pytest.mark.parametrize("method", ["bfill", "backfill", "pad", "ffill"])
 def test_reindex_index_datetime_with_fill(limit, method):
     date_index = native_pd.date_range("1/1/2010", periods=6, freq="D")
     native_series = native_pd.Series(
-        {"prices": [100, 101, np.nan, 100, 89, 88]}, index=date_index
+        {"1/1/2020": [100, 101, np.nan, 100, 89, 88]}, index=date_index
```

Comment:
let's keep the tests with xfail and add a todo.

Reply:
done!